spec/index.bs

```diff
@@ -1282,6 +1288,7 @@ interface GPUSupportedLimits {
     readonly attribute unsigned long maxInterStageShaderComponents;
     readonly attribute unsigned long maxComputeWorkgroupStorageSize;
     readonly attribute unsigned long maxComputeWorkgroupInvocations;
+    readonly attribute GPUExtent3D maxComputeWorkgroupSize;
```

Comment:
Using the `GPUExtent3D` is a bit strange here. It's made for texture sizes specifically, hence `depthOrArrayLayers` field, but the compute workgroup grid has no direct relation to the texture sizes.

Reply:
It is readonly and described as `[256, 256, 64]` above. The intent is that it is always the array version. I agree that the parallels with texture size are weird because they should be non-existent, but we're really talking about a volume here so it makes sense to use `Extent3D` imho.

Reply:
If the intent is to use the array, any reason to not put `sequence<GPUIntegerCoordinate>` in here?
I don't know how users are expected to deal with `GPUExtent3D` coming from *us*. It's one thing to pass in either a list or a structure. It's another thing to not know what to do with this type when it comes *to* you.

Reply:
The reason is because WebIDL doesn't have `[GPUIntegerCoordinate; 3]` to show the sequence is statically sized, while `GPUExtent3D` is clearly sized. But happy to change, this doesn't really matter.

Reply:
Yeah, no strong feelings here, we can follow-up